druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -3242,6 +3244,36 @@ private void testCountWithApproxDistinct(boolean approx, String sql, String expe
         .returnsUnordered("A=225541.91732536256");
   }
 
+  @Test public void testIsNull() {
+    final String sql = "select min(\"store_cost\") as a "
+        + "from \"foodmart\" "
+        + "where \"product_id\" is null";
+    final String druidQuery =
+        "{'queryType':'timeseries','dataSource':'foodmart','descending':false,'granularity':'all',"
+            + "'filter':{'type':'selector','dimension':'product_id','value':null},"
+            + "'aggregations':[{'type':'doubleMin','name':'A','fieldName':'store_cost'}],"
+            + "'intervals':['1900-01-09T00:00:00.000/2992-01-10T00:00:00.000'],"
+            + "'context':{'skipEmptyBuckets':true}}";
+    sql(sql, FOODMART)
+        .queryContains(druidChecker(druidQuery))
+        .returnsCount(0);
+  }
+
+  @Test public void testIsNotNull() {
```

Comment:
I guess this answers to https://github.com/apache/calcite/pull/546/files#r143879698